src/SDKs/ServiceBus/data-plane/README.md

```diff
@@ -0,0 +1,173 @@
+# Microsoft Azure SDK for .NET
```

Comment:
This does not seem to follow our REDME format. In particular, we wanted our readmes to start with how to use the package and then much later on talk about how to contribute to the package.

Reply:
I was requested to keep as much as-is as possible for the initial adoption.  This was the ReadMe as it exists in the source repository.  

@AlexGhiondea - Thoughts?  Should we rewrite to snap to local norms or keep with the minimal change approach?

Reply:
This readme does feel more like a how to build/contribute to this library as opposed to how to consume this library also a lot of it applies to more then just this library so I'm with @KrzysztofCwalina that this should be updated but I don't have concerns if it happens as a secondary step.

Reply:
@jsquire let's file an issue to update the readme after we merge this.

Reply:
Will do.  Taking a note to add it and resolving.

Reply:
Created:  (Service Bus Client: Enhance ReadMe #5349)[https://github.com/Azure/azure-sdk-for-net/issues/5349]